documentation/target-versions.csv

```diff
@@ -1,4 +1,5 @@
 "``mmif-python`` version","Target MMIF Specification"
+1.0.9.dev1,"1.0.0"
```

Comment:
This line is added only as a development stage, and shouldn't be included in the commit at all.